charters/DeiCharter.md

```diff
@@ -0,0 +1,67 @@
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.
+
+## WHY: Purpose 
+
+Why is the committee being created? How does this group contribute to Jupyter? 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
+
+## WHAT: Mission/Areas of Responsibility
+
+What will this committee work on? How will the purpose described above be accomplished?
+
+The DEI standing committee is responsible for: 
+* Propose, execute, collect metrics, and evaluate the impact of DEI initiatives
+* Serve as a point of contact for DEI topics for the global Jupyter community
+* Serve as DEI ambassadors by including this topic in the discussions and spaces within Project Jupyter
+* Recommend and promote training, educational, and motivational materials on DEI 
+* Flag DEI issues or potential issues to the Executive Council (EC) or the Software Steering Council (SSC)
+
+## WHO: Committee Composition
+### Members
+Members should affirm the opening paragraph of this charter and agree to the purpose of this committee
+The committee should consist of at least 2 members serving a 1 year term with an option for unlimited renewal 
+### General Members
+Responsibilities:
+* Attend meetings as scheduled by the vice president/co-chair(s)
+* Complete onboarding tasks in a timely manner
+* Assist in carrying out the committee’s objectives
+* Adding new members
+* An existing member may nominate a new member for the committee
+* Nominations are discussed at committee meetings;  nominations are accepted by a simple majority 
+* In extraordinary circumstances, the Chair may reject the nomination
+### Leadership Members:
+For smaller committees (8 or less), only a chair is required. For large committees (9 or more), three leadership positions are required. One person may hold a leadership position and also serve as an SSC rep.
+#### Chair
+* Board appointed or elected by committee members for a one-year term with an option to serve an additional one-year term upon member approval.
+* May be submitted to the Executive Council for approval as part of a new committee proposal (one-year term)
+* Responsibilities:
+  * Calls for scheduled meetings
+  * Sets the agenda for meetings
+  * Enforces Jupyter’s Code of Conduct as needed
```

Comment:
I would remove this CoC bullet here and under vice-chair if Jupyter is going to have a Code of Conduct standing committee so it is clear who is responsible for Code of Conduct issues and remediation.

Perhaps change to something along the lines of:
- Provide feedback and insights to the Code of Conduct committee

Reply:
We actually were already planning to remove the "Who" section - just pushed the update without that section